### ISSUE TITLE
How to enable keyboard on select items

### ISSUE THREAD

**Issue by krugerdavid** [paliari/v-autocomplete#15]
Hi!
This really is a great component. But is there a way to use the keyboard when the list is displayed? instead of clicking, just selecting it with the arrow key
Thanks!

**Comment by paliari** [comment]
thanks for the tip.
implemented in version 1.5.0.

to change the appearance of the active item, just overwrite the css:
```
  .v-autocomplete .v-autocomplete-list .v-autocomplete-list-item.v-autocomplete-item-active {
    background-color: gray;
  }
```

**Comment by krugerdavid** [comment]
@paliari is not about the appearance.. is when for example I start typing a state name and the can select with the down arrow.. 
Regards

**Comment by dflourusso** [comment]
@krugerdavid Your suggestion was implemented in version 1.5.0, please update you package.
You can test in the [DEMO](https://paliari.github.io/v-autocomplete/)

**Comment by krugerdavid** [comment]
ohhh thanks!! wasn't sure if I understood correctly! thanks @paliari @dflourusso
